core/common/util/src/testFixtures/java/org/eclipse/dataspaceconnector/common/util/junit/annotations/ApiTest.java

```diff
@@ -0,0 +1,32 @@
+/*
+ *  Copyright (c) 2022 Microsoft Corporation
+ *
+ *  This program and the accompanying materials are made available under the
+ *  terms of the Apache License, Version 2.0 which is available at
+ *  https://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  SPDX-License-Identifier: Apache-2.0
+ *
+ *  Contributors:
+ *       Microsoft Corporation - initial API and implementation
+ *
+ */
+
+package org.eclipse.dataspaceconnector.common.util.junit.annotations;
+
+import org.junit.jupiter.api.Tag;
+
+import java.lang.annotation.ElementType;
+import java.lang.annotation.Retention;
+import java.lang.annotation.RetentionPolicy;
+import java.lang.annotation.Target;
+
+/**
+ * Common annotation for integration testing.  It applies integration-test Junit Tag.
+ */
+@Target({ ElementType.TYPE })
+@Retention(RetentionPolicy.RUNTIME)
+@IntegrationTest
+@Tag("ApiTest")
+public @interface ApiTest {
```

Comment:
I was wondering... could these test annotation stay in the `junit` module instead of the abused "common-util"? Seems like a better place for them

Reply:
It make sense, 
probably it make sense also to move that module in core instead of extensions?

I see a lot of core modules that depends on `testImplementation(project(":extensions:common:junit"))` for extension testing

Reply:
That could be good, but in case I'd open an issue for that to gather some opinions